src/wazuh_modules/vulnerability_scanner/testtool/scanner/main.cpp

```diff
@@ -76,47 +76,50 @@ int main(const int argc, const char* argv[])
             configuration);
 
         // Wait for the complete initialization and connection negotiation.
-        std::this_thread::sleep_for(std::chrono::seconds(1));
+        std::this_thread::sleep_for(std::chrono::seconds(60));
```

Comment:
This time might not be enough for some situations, or it might be too much in others.
Shouldn't the tool wait for some event?

Reply:
It was moved in case you just wanted to download the content. This is enough time while the snapshot processing starts.